share/sysbuild/images/bootloader/Kconfig

```diff
@@ -26,6 +26,13 @@ config BOOTLOADER_MCUBOOT
 	help
 	  Include MCUboot (Zephyr port) as the bootloader to use
 
+config BOOTLOADER_CUSTOM
+	bool "Custom bootloader"
+	help
+	  Include custom bootloader in the build
+	  (the bootloader build must be configured and triggered
+	  from the target SOC or BOARD sysbuild system)
```

Comment:
You should add this to your soc file and not call it custom, call it what you want to call it

Reply:
Removed